addons/bestja_page_fixtures_ucw/templates/pages/home_page.xml

```diff
@@ -255,11 +288,11 @@
                             </div>
                         </div>
                     </section>
-                    <section class="oe_snippet_body join_us_now">
+                    <section class="oe_snippet_body">
                         <div class="container">
                             <div class="row text-center mt64 mb64">
                                 <div class="col-md-12">
-                                    <a class="btn btn-lg mt32 mb32 text-center link_like_button" t-if="website.user_id == user_id" href="/web/signup">DOŁĄCZ TERAZ</a>
+                                    <a class="btn btn-lg mt32 mb32 text-center link_like_button" t-if="website.user_id == user_id" href="/web/signup">DOŁĄCZ TERAZ</a><br />
```

Comment:
I am sorry, I can't agree, Odoo server gives error like:
```XMLSyntaxError: Opening and ending tag mismatch: br line 295 and div, line 297, column 39```
if there is no slash in ```<br>``` tag.